service/gcs/core/gcs/storage.go

```diff
@@ -266,7 +262,7 @@ func (c *gcsCore) mountMappedDirectories(dirs []prot.MappedDirectory) error {
 		if !dir.CreateInUtilityVM {
 			return errors.New("we do not currently support mapping directories inside the container namespace")
 		}
-		mountedPath := filepath.Join(mappedDirectoryPrefix, dir.ContainerPath)
+		mountedPath := dir.ContainerPath
```

Comment:
This was honestly just because `mountedPath` is more concise than `dir.ContainerPath`, and strings in Go are just pointers and lengths, so the copy should be pretty inexpensive. If we want, I can just use dir.ContainerPath/disk.ContainerPath directly in storage.go, though. I don't particularly care either way.